plugins/catalog-backend-module-github/src/lib/github.ts

```diff
@@ -106,22 +124,24 @@ export async function getOrganizationUsers(
             bio,
             email @include(if: $email),
             login,
-            name
+            name,
+            organizationVerifiedDomainEmails(login: $org)
           }
         }
       }
     }`;
 
   // There is no user -> teams edge, so we leave the memberships empty for
   // now and let the team iteration handle it instead
-  const mapper = (user: User) => {
+
+  const defaultUserMapper: IMapper<User, UserEntity> = (item: User) => {
```

Comment:
You should define this default mapper on the root level (maybe even in a file of its own) and exporting it all the way out so it gets visible in api-report.md as `defaultUserTransformer`